src/main/java/com/ita/if103java/ims/exception/EntityNotFoundException.java

```diff
@@ -0,0 +1,23 @@
+package com.ita.if103java.ims.exception;
+
+public class EntityNotFoundException extends RuntimeException {
+
+    private String name;
+
+    public EntityNotFoundException(String message) {
+        super(message);
+    }
+
+    public EntityNotFoundException(String message, String name) {
+        super(message);
+        this.name = name;
+    }
+
+    public String getName() {
+        return name;
+    }
+
+    public void setName(String name) {
+        this.name = name;
+    }
+}
```

Comment:
I think that we can to create our `BaseImsRuntimeException` like this:
```java
public abstract class BaseImsRuntimeException extends RuntimeException {

    private String name;

    public BaseImsRuntimeException(String message) {
        super(message);
    }

    public BaseImsRuntimeException(String message, String name) {
        super(message);
        this.name = name;
    }

    public String getName() {
        return name;
    }

    public void setName(String name) {
        this.name = name;
    }
}
```
Then use it without code duplication:
```java
class CRUDException extends BaseImsRuntimeException {
 
    public CRUDException(String message) {
        super(message);
    }

    public CRUDException(String message, String name) {
        super(message, name);
    }
}
```

```java
class EntityNotFoundException extends BaseImsRuntimeException {
   
    public EntityNotFoundException(String message) {
        super(message);
    }

    public EntityNotFoundException(String message, String name) {
        super(message, name);
    }
}
```